### PR TITLE
Update to yarn package manager

### PR DESCRIPTION
**Pull request by ssivov**

### CODE REVIEW

**Comment by ender336** [comment]
update our contributing.md to reflect the new system please

**Comment by ssivov** [comment]
I was planning a separate docs overhaul, but i'll include minimal changes in here

---
In reply to: [457772997](https://github.com/Microsoft/clarity-js/pull/129#issuecomment-457772997) [](ancestors = 457772997)

File: package.json [hunk 1]
```diff
@@ -58,18 +58,23 @@
...
+    "test:firefox": "karma start karma/configs/firefox.js",
+    "test:ie": "karma start karma/configs/ie.js",
+    "tslint": "tslint --project ./",
+    "tslint:fix": "tslint --fix --project ./ --force"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
   },
```
Comment (ssivov) [inline package.json]: This is unrelated to yarn, but husky has been complaining about outdated "precommit" declaration, so sneaking in a quick fix for that.

File: src/converters/schema.ts [hunk 2]
```diff
@@ -92,5 +92,6 @@ export function resetSchemas() {
   schemaManager.reset();
 }
 
+// More changes
```
[the remark this answers is not in the document]
Comment (ssivov) [reply src/converters/schema.ts]: Woops. Fixed

---
In reply to: [251175319](https://github.com/Microsoft/clarity-js/pull/129#discussion_r251175319) [](ancestors = 251175319)

**Pull request merged**